### PR TITLE
Feature/us022 enter enrolment code

### PR DESCRIPTION
**Pull request by duncannixon**
US022 Enter Enrolment Code
US150 Confirm org and survey
Ras common integration
RAD Iterations

Updated unit tests to follow.

### CODE REVIEW

**Comment by oddjobz** [comment]
Should probably fix the tests tho' before a merge ...

**Comment by oddjobz** [comment]
Merged with the caveat that the next job on the list is fixing the unit tests .. ;)

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+git+https://github.com/ONSdigital/ras-common.git@US032b/case-service-logging
```
Comment (rjweeks) [inline requirements.txt]: This is a feature branch. Should this change be merged into master?

Comment (duncannixon) [inline requirements.txt]: This is part of the integration piece and is required for the time being as the feature branch being referenced is still in the PR column. I will update this once the PR has been merged to master.

**Pull request merged**